website/package.json

```diff
@@ -72,6 +72,7 @@
     "babylon6": "npm:babylon@^6",
     "babylon7": "npm:@babel/parser@^7",
     "cherow": "^1.6.8",
+    "chevrotain": "^7.0.1",
```

Comment:
If this is installed as a dependency of `java-parser`, this shouldn't really be here...

Reply:
Removed 👍